.github/actions/nodejs/action.yaml

```diff
@@ -17,6 +17,11 @@ runs:
       shell: bash
       run: npm config set audit false
 
+    - uses: aws-actions/configure-aws-credentials@v4
+      if: contains(runner.name, 'ubuntu-platform')
+      with:
+        aws-region: eu-west-1
+
     - name: Cache NPM build artifacts
```

Comment:
We still keep it? Shall we use action/cache if aren't in AWS?

Reply:
Good point, added a condition here as well